app/scripts/components/exploration/analysis-data.ts

```diff
@@ -236,7 +236,7 @@ export async function requestCMRTimeseriesData({
       ? datasetData.tileApiEndpoint.replace('WebMercatorQuad/tilejson.json', '')
       : envApiCMREndpoint;
     const statResponse = await queryClient.fetchQuery(
-      ['analysis', datasetData.id, 'cmr', aoi],
+      ['analysis', datasetData.id, 'cmr', aoi, start, end],
```

Comment:
I notice we don't use the formattedParamString in the query key. This leads me to believe to all params should cache bust this query. In that case, are there any additional params I should add?

`datasetData.timeInterval`?